addons/mail/static/src/discuss/core/web/discuss_command_palette_patch.js

```diff
@@ -13,6 +13,12 @@ commandCategoryRegistry
     .add(DISCUSS_RECENT, { namespace: "@", name: _t("Recent") }, { sequence: 20 });
 
 patch(DiscussCommandPalette.prototype, {
+    async fetch() {
+        await this.store.channels.fetch();
+        if (this.cleanedTerm) {
+            await this.store.searchConversations(this.cleanedTerm);
+        }
+    },
```

Comment:
Better have minimal patch and reuse code, such as defining a `if (shouldSearch)` function that is `true` originally and patched in `web`.